### PR TITLE
Fix delete not found image

### LINKED ISSUE #76: Unsaved deleted images get stuck
If you delete an image in an editor but don't save that post the reference is still saved on the post but points to nothing. You can then not delete the reference as the delete function errors when the target is not found.

### PR DESCRIPTION
**Pull request by mixcas**
Fix deleting an image that for w/e reason has been removed from the storage but there's still records pointing to it in the db

closes #76

### CODE REVIEW

File: src/utilities/images.js [hunk 1]
```diff
@@ -37,10 +37,8 @@ export const getResizedImageUrl = (file, size, square) => {
     return;
   }
 
-  const extension = file.name.split('.').pop();
```
Comment (mixcas) [inline src/utilities/images.js:40]: Not used

File: src/components/obras/ObrasGroup.jsx [hunk 2]
```diff
@@ -81,7 +81,7 @@ class ObrasGroup extends Component {
 
     const id = event.target.options[event.target.selectedIndex].value;
 
-    const { value: { artista, medium, title, year, materials, tecnica, images } } = this.props.allObras.find( obra => obra.key === id );
```
Comment (mixcas) [inline src/components/obras/ObrasGroup.jsx:84]: `medium` wasn't used anymore

File: src/components/obras/ObraForm.jsx [hunk 3]
```diff
@@ -15,7 +15,6 @@ import { Editor } from 'react-draft-wysiwyg';
 
 import 'react-draft-wysiwyg/dist/react-draft-wysiwyg.css';
 
-import Uploads from '../fields/Uploads';
```
Comment (mixcas) [inline src/components/obras/ObraForm.jsx:18]: Not used

File: src/components/fields/ImageUploadsImage.jsx [hunk 4]
```diff
@@ -2,7 +2,6 @@ import React, { Component } from 'react';
 import { toastr } from 'react-redux-toastr';
 
 import { ToastrOptionsConfirm } from '../../utilities/toastr.js';
-import { getResizedImageUrl } from '../../utilities/images.js';
```
Comment (mixcas) [inline src/components/fields/ImageUploadsImage.jsx:5]: Not used

**Comment by varffs** [comment]
You missed a lint :(

**Comment by mixcas** [comment]
I couldn't fix it :(

**Pull request merged**